src/main/webapp/src/app/app.module.ts

```diff
@@ -18,6 +18,23 @@ import {HttpClientModule} from '@angular/common/http';
 import {RouterModule} from '@angular/router';
 
 import {AppRoutingModule} from './app-routing.module';
+//import {MaterialComponents} from './common/material-components';
```

Comment:
nit: before submitting please remove all commented out imports

Reply:
Thanks for reminding me, I forgot that I commented anything in this file when I did my checks.